opennebula/shared_schemas.go

```diff
@@ -106,6 +106,7 @@ func diskFields(customFields ...map[string]*schema.Schema) map[string]*schema.Sc
 		"volatile_format": {
 			Type:        schema.TypeString,
 			Optional:    true,
+			Default:     "raw",
```

Comment:
Can you please update the description to precise the default value ? thank you

Reply:
Sorry I missed that one. Updated.